roles/create_certification_project/tasks/validate_cert_settings.yml

```diff
@@ -15,7 +15,30 @@
       - "cert_settings.repository_description is defined"
     fail_msg: "The parameter {{ item }} is required to update cert project"
     success_msg: "The parameter {{ item }} is present"
-  when: product_type == 'container'
+  when:
+    - product_type == 'container'
+    - do_container_parallel_test | bool
+
+- name: Validate cert_settings for container project
+  ansible.builtin.assert:
+    that: "{{ item }} is defined"
```

Comment:
I believe it's better to assign a custom variable to the `item` in order to validate the assertion

Reply:
maybe something like this: `loop_var: container_setting`?
```
- name: Validate cert_settings for container project
  ansible.builtin.assert:
    that: "{{ container_setting }} is defined"
    fail_msg: "The parameter {{ container_setting }} is required to update cert project"
    success_msg: "The parameter {{ container_setting }} is present"
  loop:
    - cert_item.short_description
    - cert_item.short_description | length >= 50
    - cert_settings.auto_publish
    - cert_settings.email_address
    - cert_settings.build_categories
    - cert_settings.registry_override_instruct
    - cert_settings.application_categories
    - cert_settings.os_content_type
    - cert_settings.privileged
    - cert_settings.release_category
    - cert_settings.repository_description
  loop_control:
    loop_var: container_setting
  when:
    - product_type == 'container'
    - not do_container_parallel_test | bool
```

Reply:
we somehow replaced the `item` var, not that in play exactly, please see commit: 775294b91fb6bcbf551db53c33bdaceb7a29c45f